odl/contrib/mrc/README.md

```diff
@@ -0,0 +1,21 @@
+# mrc
```

Comment:
capital letters, also this title should be larger than others

Also, we really should add readme's like this to different places. The prime examples would be the examples folders.